fpdf/text_region.py

```diff
@@ -631,6 +632,7 @@ def __init__(
         self.balance = balance
         total_w = self.extents.right - self.extents.left
         col_width = (total_w - (ncols - 1) * gutter) / ncols
+        self.h_margins = (pdf.c_margin, pdf.c_margin)
```

Comment:
Given the current usage of this attribute, I wonder if having 2 distincts attributes `.l_margin` & `.r_margin` would not make the code & intent clearer there? 🙂